_1327/information_pages/views.py

```diff
@@ -22,8 +22,9 @@ def unlinked_list(request):
 				except ValueError:
 					pass
 
-		unlinked_information_pages = get_objects_for_user(request.user, permission_name, klass=InformationDocument.objects.filter(menu_items__isnull=True).exclude(id__in=menu_page_document_ids)).order_by('title')
-
+		title_string = "title_de" if get_language() == "de" else "title_en"
```

Comment:
this might have the same problem we encountered in the `translate` function... I think the same solution could be applied here.